b2b-sample/guardio-insurance-sample-application/components/settingsComponents/identity-providers/idpDetailsSections/general.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import { Field, Form } from 'react-final-form';
+import { Button, ButtonToolbar, Loader, useToaster } from 'rsuite';
+import FormSuite from 'rsuite/Form';
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from '../../../../util/util/frontendUtil/frontendUtil';
+import { errorTypeDialog, successTypeDialog } from '../../../util/dialog';
+
+import styles from '../../../../styles/Settings.module.css';
+import decodePatchGeneralSettingsIdp from
+    '../../../../util/apiDecode/settings/identityProvider/decodePatchGeneralSettingsIdp';
+import HelperText from '../../../util/helperText';
+import { checkIfJSONisEmpty } from '../../../../util/util/common/common';
+
+export default function General(props) {
+
+    const [loadingDisplay, setLoadingDisplay] = useState(LOADING_DISPLAY_NONE);
+
+    const toaster = useToaster();
+
+    const nameValidate = (name, errors) => {
+        if (!name) {
+            errors.name = 'This field cannot be empty'
+        }
+
+        return errors;
+    }
+
+    const descriptionValidate = (description, errors) => {
+        if (!description) {
+            errors.description = 'This field cannot be empty'
+        }
+
+        return errors;
+    }
+
+    const validate = values => {
+        let errors = {}
+        errors = nameValidate(values.name, errors);
+        errors = descriptionValidate(values.description, errors);
+        
+        return errors
```

Comment:
add a `;` in return statement.

Reply:
This was after adding the eslint in the `identity-apps` to the project.

### Commit
- https://github.com/wso2/samples-is/commit/e50bfd37f9fdd026c3f685b6fe602f7fdcc542e8#diff-6e806d4538a673d7ea75a914962e3f09e15ecbb1754339a574f54c5c619636d7